.github/workflows/linux-nightly.yml

```diff
@@ -54,3 +55,47 @@ jobs:
           for pkg in fixed_dist/torcharrow*.whl; do
               ~/.local/bin/aws s3 cp "$pkg" "$S3_PATH" --acl public-read
           done
+
+  macos-container:
+    runs-on: macos-11
+    strategy:
+      fail-fast: true
+      matrix:
+        python-version:
+          - 3.7
+          - 3.8
+          - 3.9
+    steps:
+      - name: Setup Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Check out source repository
+        uses: actions/checkout@v2
+        with:
+            submodules: true
+
+      - name: Build the wheel
+        run: |
+          ./csrc/velox/velox/scripts/setup-macos.sh
```

Comment:
unrelated to this diff; But would be great for Velox repo to provide and maintain the "setup-{macos/ubuntu}-minimal.sh"

Reply:
Yes makes sense . Will do that as a part of kr3.